src/api/main.py

```diff
@@ -0,0 +1,65 @@
+import os
+from os.path import join
+import errno
+import shutil
+from subprocess import call
+import json
+import uuid
+
+from flask import Flask, jsonify
+
+from rwd.Rapid_Watershed_Delineation import Point_Watershed_Function
+
+app = Flask(__name__)
+
+
+@app.route('/rwd/<lat>/<lon>', methods=['GET'])
+def run_rwd(lat, lon):
```

Comment:
I'm thinking the snapping vars should be part of a query string, but I can add them to the URL structure if you think I should.